.npmrc

```diff
@@ -1 +1,2 @@
 auto-install-peers=false
+resolution-mode=highest
```

Comment:
This resolution mode makes more sense to me than the default (oldest). Another way would could handle this is much more strict versions in our `package.json` files.